video_xblock/static/html/brightcove.html

```diff
@@ -18,7 +18,7 @@
         data-embed="default"
         data-application-id
         data-setup='{"playbackRates": [0.5, 1, 1.5, 2] }'
-        width="829"
+        width="100%"
         height="511"
```

Comment:
If this is not possible, add a short comment that it's value doesn't matter and will be calculated at rendering time.